README.md

```diff
@@ -79,6 +79,11 @@ pip install -e .
 python -m streamlit run demo/app.py
 ```
 
+## System requirements
+  - OS: Windows, macOS, or Linux
+  - Python 3.10+ / 3.12+ for Apple M chips
+  - Minimum RAM: 8 GB
+  - Disk space: 20 GB minimum
```

Comment:
Where are the 20GB coming from?

Reply:
Well I thought `venv` is ~7GB, `bartowski/Qwen2.5-7B-Instruct-GGUF/Qwen2.5-7B-Instruct-Q8_0.gguf` is also ~8Gb, kokoro is lets say ~1GB, and then some slack space for loading the doc and stuff? :sweat_smile: As you see my method is very scientific... do you have any other suggestions?